packages/patternfly-3/patternfly-react-wooden-tree/src/components/TreeView/index.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { Tree } from 'react-wooden-tree';
+
+require('react-wooden-tree/dist/react-wooden-tree.css');
+require('./style.css');
+
+const data = [
```

Comment:
Uh oh, sorry that I missed this before, but this hardcoded data should probably get moved out of this file.  Perhaps for now, you could create a file

```
patternfly-react-wooden-tree/src/components/TreeView/TreeView.fixtures.js
```
to be used for the storybook.

Looking at the [npm documentation](https://www.npmjs.com/package/react-wooden-tree), I think we'll want to pass the tree data down to <TreeView> as a prop